internal/proto/controller/api/services/v1/scope_service.proto

```diff
@@ -153,3 +166,10 @@ message ListKeysRequest {
 message ListKeysResponse {
   repeated resources.scopes.v1.Key items = 1;
 }
+
+message RotateKeysRequest {
+  string id = 1;
```

Comment:
Maybe make this `scope_id` since it's going into the request body rather than the path?